### DISCUSSION TITLE
My vigitation is not open. I try many times. Error show on screen.

### DISCUSSION THREAD

**Discussion by chyasirabbasjutt** [cu-esiil-edu/2024-data-short-course#82] [Q&A]
![image](https://github.com/cu-esiil-edu/2024-data-short-course/assets/166647114/e8726cb2-90cd-405f-a040-1ce5f77503f9)
![image](https://github.com/cu-esiil-edu/2024-data-short-course/assets/166647114/04576d3e-c980-422a-bc18-da779d24c326)
please help me @nquarder @eculler

**Comment by nquarder** [comment]
hi @chyasirabbasjutt it looks like for the 2nd screen shot from the open/reproducible science notebook. it looks like you're running into an issue saving to html because you are giving it a different file name. Your file name is:

```
get-started-with-open-reproducible-science!.ipynb 

```

Notice the exclamation point ! at the end? That is missing in the cell where you convert from ipynb to html


As for the issue about the vegetation notebook, I am less sure about that one but we can try to help during office hours if you are able to join here in a bit.

**Comment by nquarder** [comment]
hi @chyasirabbasjutt - I have been playing with some NDVI data in Pakistan (FATA) and have a notebook to share with you to review. I also added some fire boundaries for 2021. They are so small it is hard to see a difference in NDVI inside/outside the fire perimeter. There are some interesting patterns though, for example, I think you can make out the Indus River east of the FATA boundary. 

Check this out and would love to talk more about this area and findings: https://nquarder.github.io/notebooks/2021-pakistan-fires.html
